source/templates.tex

```diff
@@ -5148,9 +5111,9 @@
 \pnum
 A class member access expression\iref{expr.ref} is
 type-dependent if
+the terminal name of its \grammarterm{id-expression}, if any, is dependent or
```

Comment:
"is dependent or" -> "is dependent, or"

Reply:
Adding a comma risks confusing levels. I'm good without the comma; the two phrases are very short.

Reply:
Let's use bullets then.  As it is, we have "X or Y and Z"; it's not possible to tell if this is supposed to be interpreted as "(X or Y) and Z" or "X or (Y and Z)".

Reply:
deferred